spec/views/post_user_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+RSpec.describe 'User Posts', type: :feature do
+  before do
+    @user = User.create!(name: 'Lulu', posts_counter: 0)
+    @post1 = Post.create!(title: 'Post 1', text: 'This is post 1', author: @user, comments_counter: 0, likes_counter: 0)
+    @post2 = Post.create!(title: 'Post 2', text: 'This is post 2', author: @user, comments_counter: 0, likes_counter: 0)
+    @post3 = Post.create!(title: 'Post 3', text: 'This is post 3', author: @user, comments_counter: 0, likes_counter: 0)
+    @comment = Comment.create!(text: 'Nice post', user: @user, post: @post1)
+
+    visit user_posts_path(@user)
+  end
+
```

Comment:
- [ ] I can see tests only for the `posts#index` and `posts#show` pages, You missed adding tests for users' pages. Please make sure to add all tests as requested [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=User%20index%20page,post%27s%20index%20page) in the project requirements.